components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -129,7 +129,16 @@ - (void)addActionButton:(nonnull MDCButton *)button {
       // reset the title to the default
       [button setTitleColor:_buttonColor forState:UIControlStateNormal];
     }
-    [button setTitleFont:_buttonFont forState:UIControlStateNormal];
+    if (button.enableTitleFontForState) {
+      if (![button titleFontForState:UIControlStateNormal]) {
+        [button setTitleFont:[MDCAlertControllerView buttonFontDefault]
+                    forState:UIControlStateNormal];
+      }
+    } else {
+      if (!button.titleLabel.font) {
```

Comment:
This can be flattened to:

```objc
    } else if (!button.titleLabel.font) {
```

Reply:
Why are we checking this instead of simply overriding the existing font, like in the other places?